CMakeLists.txt

```diff
@@ -3,15 +3,12 @@ cmake_minimum_required(VERSION 2.8.3)
 project(rclpy)
 
 find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
 find_package(rcl REQUIRED)
 find_package(rmw REQUIRED)
 find_package(rmw_implementation_cmake REQUIRED)
-find_package(ament_cmake_python REQUIRED)
-
-include_directories(include)
 
 if(NOT WIN32)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
```

Comment:
If this package doesn't do any C++ stuff `project(...)` should explicitly list only `C` to avoid searching for a C++ compiler.